lms/static/sass/base/_variables.scss

```diff
@@ -166,7 +166,7 @@ $m-gray-l1: rgb(151,153,155); // #97999B
 $m-gray-l2: rgb(164,166,168); // #A4A6A8
 $m-gray-l3: rgb(177,178,180); // #B1B2B4
 $m-gray-l4: rgb(245,245,245); // #F5F5F5
-$m-gray-d1: rgb(125,127,131); // #7D7F83
+$m-gray-d1: rgb(111, 112, 116); // #6f7074 AA compliant
```

Comment:
I see the value in noting combinations that have been checked and are compliant, the challenge here is that this is just one variable. How its used or what its used with can change - the case you revised against a lighter gray BG might not always be how/where this variable is referenced. Saying its compliant with that concern in mind means that comment can be confusing or misunderstood.

Perhaps moving that comment to where we're using this variable in that case you've solved is the better answer and we can start to note AA/AAA compliancy in application cases/rules. Sound okay?